plutus-cbor/src/Plutus/Codec/CBOR/Encoding.hs

```diff
@@ -0,0 +1,77 @@
+module Plutus.Codec.CBOR.Encoding (
+  -- * Encoding
+  Encoding,
+  encodingToBuiltinByteString,
+
+  -- * Basic types
+  encodeInteger,
+
+  -- * Data-structures
+) where
+
+import PlutusTx.Prelude
+
+-- * Encoding
+
+type Encoding = BuiltinByteString
+
+encodingToBuiltinByteString :: Encoding -> BuiltinByteString
+encodingToBuiltinByteString = id
+{-# INLINEABLE encodingToBuiltinByteString #-}
```

Comment:
What's the point of this function?

Reply:
Well, the fact that the module uses a `BuiltinByteString` as an intermediary representation of encoding is sort of an implementation detail, so, when considering the library from a user perspective, this should not transpire through the interface IMO.

Reply:
A newtype with non-exported or marked `Unsafe` data constructor would hint at that more rigorously IMO